ClassLibrary/ClassLibrary.Test/UnitTest1.cs

```diff
@@ -5,18 +5,31 @@ namespace ClassLibrary.Test
     [TestFixture]
     public class UnitTest1
     {
+        Library lib = new Library();
```

Comment:
It's not a correct approach. Each test method should have an isolated set of objects under test. Unit tests must be isolated, later we'll learn how to test complex scenarios. For now you should always create all objects under test inside the test method.